understanding/text-spacing.html

```diff
@@ -58,268 +58,199 @@
-                        Plugin technologies would need to have a built-in ability to modify styles to the
-                        specified metrics. Currently, this SC does not apply to PDF as it is not implemented
-                        using markup.
+                     <p>使用されているマークアップベースの技術が、テキストをこの達成基準の指標で上書きできる場合、この達成基準は適用可能である。例えば、Cascading Style Sheet / HTML 技術は、規定された間隔の指標を許すことが十分に可能である。プラグイン技術には、スタイルを規定された指標に変更するための組み込み機能が必要である。現在のところ、PDF はマークアップを使用して実装されていないため、この達成基準は適用されない。
                      </p>
                      
-                     <p>Examples of text that are typically not affected by <a href="#dfn-style-property">style properties</a> and not expected to adapt are:
+                     <p><a href="#dfn-style-property">スタイルプロパティ</a>の影響を受けず、適応が見込まれないテキストの例は次の通りである:
                      </p>
                      
                      <ul>
                         
-                        <li>Video captions embedded directly into the video frames and not provided as an associated
-                           caption file
+                        <li>動画のフレームに直接埋め込まれ、関連付けられたキャプションファイルとしては提供されない、動画のキャプション
                         </li>
                         
-                        <li><a href="#dfn-image-of-text">Images of text</a></li>
+                        <li><a href="#dfn-image-of-text">文字画像</a></li>
                         
                      </ul>
                      
-                     <p>For this SC, <a href="https://html.spec.whatwg.org/multipage/canvas.html#the-canvas-element">canvas</a> implementations of text are considered to be <a href="#dfn-image-of-text">images of text</a>.
+                     <p>この達成基準では、<a href="https://html.spec.whatwg.org/multipage/canvas.html#the-canvas-element">canvas</a> でのテキストの実装は<a href="#dfn-image-of-text">文字画像</a>と見なされる。
                      </p>
                      	
                   </section>
                   
                   <section id="use-of-ellipses">
                      
-                     <h4>Use of ellipses</h4>
+                     <h4>省略記号の使用</h4>
                      
-                     <p>There may be regions of a page where text containers cannot expand due to design constraints
-                        (such as a maximum width for the left navigation or table column headers). A common
-                        convention if text exceeds its space is to replace truncated text with an ellipsis.
-                        Where ellipses appear as a result of modifying text style properties, the page can
-                        still meet the Text Spacing requirements, so long as the content is still available.
-                        For example:
+                     <p>デザイン上の制約 (左ナビゲーションやテーブルの列見出しの最大幅など) で、テキストコンテナを広げることができないページ領域があるかもしれない。テキストがその間隔を超えた場合の一般的な慣例は、切り詰められたテキストを省略記号で置き換えることである。テキストスタイルプロパティを変更した結果、省略記号が表示された場合でも、コンテンツが利用可能である限り、そのページはテキストの間隔の要件を満たすとすることができる。例えば:
```

Comment:
> text exceeds its space

ここのspaceを「間隔」と取ってしまうと、おかしなことになるような？（テキストコンテナを超えてしまったときのことを言っているので）

```suggestion
                     <p>デザイン上の制約 (左ナビゲーション、テーブルの列見出しの最大幅など) で、テキストコンテナを広げることができないページ領域があるかもしれない。テキストがその領域を超えた場合の一般的な慣例は、切り詰められたテキストを省略記号で置き換えることである。テキストスタイルプロパティを変更した結果、省略記号が表示された場合でも、コンテンツが利用可能である限り、そのページはテキストの間隔の要件を満たすとすることができる。例えば:
```